local-cli/runAndroid/runAndroid.js

```diff
@@ -27,7 +27,7 @@ function checkAndroid(root) {
  */
 function runAndroid(argv, config, args) {
   if (!checkAndroid(args.root)) {
-    console.log(chalk.red('Android project not found. Maybe run react-native android first?'));
+    console.log(chalk.red('Android project not found. Maybe run react-native eject first?'));
```

Comment:
I just tried and `react-native android` does not work with a project created with `react-native init` but `react-native eject` work well.